pyatlan/client/group.py

```diff
@@ -96,8 +96,9 @@ def get(
         limit: Optional[int] = 20,
         post_filter: Optional[str] = None,
         sort: Optional[str] = None,
-        count: bool = True,
+        count: bool = False,
```

Comment:
This was changed to False because we are not retrieving the number of filtered records or the total number of records in our result response. Instead, we are directly outputting the records. Since the default value for this setting is True, it would have impacted the test cases, leading to inconsistencies. Changing it to False ensures the expected behavior aligns with our current implementation.

Reply:
I'm not following. I see that we are populating `total_records` attribute of `GroupResponse` from the `totalRecord` field in the returned JSON

Reply:
My bad. Apologies for the confusion earlier. I've reverted it back to True. Thanks for pointing that out!